packages/bezier-react/src/components/AlphaButton/Button.module.scss

```
@@ -0,0 +1,281 @@
+      dark-grey: var(--alpha-color-bg-grey-darkest),
+      light-grey: var(--alpha-color-bg-black-dark),
+    );
+
+    @each $color, $background-color in $background-color-by-color {
+      &:where(.color-#{$color}) {
+        background-color: $background-color;
+      }
+    }
+  }
+
+  &:where(.variant-secondary) {
+    $background-color-by-color: (
+      blue: var(--alpha-color-primary-bg-lightest),
+      cobalt: var(--alpha-color-accent-bg-lightest),
+      red: var(--alpha-color-critical-bg-lightest),
+      orange: var(--alpha-color-warning-bg-lightest),
+      green: var(--alpha-color-success-bg-lightest),
+      pink: var(--alpha-color-bg-pink-lightest),
+      purple: var(--alpha-color-bg-purple-lightest),
+      dark-grey: var(--alpha-color-bg-grey-lighter),
+      light-grey: var(--alpha-color-bg-black-lighter),
+    );
+
+    @each $color, $background-color in $background-color-by-color {
+      &:where(.color-#{$color}) {
+        background-color: $background-color;
+      }
+    }
+  }
+
+  &:where(.variant-tertiary) {
+    background-color: initial;
+  }
+
+  /* color */
+  /* stylelint-disable-next-line no-duplicate-selectors */
+  &:where(.variant-primary) {
+    &:where(:not(.color-dark-grey, .color-light-grey)) {
+      color: var(--alpha-color-fg-absolute-white-dark);
+    }
+
+    &:where(.color-dark-grey) {
+      color: var(--alpha-color-fg-white-normal);
+    }
+
+    &:where(.color-light-grey) {
+      color: var(--alpha-color-fg-absolute-white-normal);
+    }
+  }
+
+  &:where(.variant-secondary, .variant-tertiary) {
+    $color-map: (
+      blue: var(--alpha-color-primary-fg-normal),
+      cobalt: var(--alpha-color-accent-fg-normal),
+      red: var(--alpha-color-critical-fg-normal),
+      orange: var(--alpha-color-warning-fg-normal),
+      green: var(--alpha-color-success-fg-normal),
+      pink: var(--alpha-color-fg-pink-normal),
+      purple: var(--alpha-color-fg-purple-normal),
+      dark-grey: var(--alpha-color-fg-black-darkest),
+      light-grey: var(--alpha-color-fg-black-darker),
+    );
+
+    @each $button-color, $color in $color-map {
+      &:where(.color-#{$button-color}) {
+        color: $color;
+      }
+    }
+
+    $icon-color-map: (
+      blue: var(--alpha-color-primary-fg-normal),
+      cobalt: var(--alpha-color-accent-fg-normal),
+      red: var(--alpha-color-critical-fg-normal),
+      orange: var(--alpha-color-warning-fg-normal),
+      green: var(--alpha-color-success-fg-normal),
+      pink: var(--alpha-color-fg-pink-normal),
+      purple: var(--alpha-color-fg-purple-normal),
+      dark-grey: var(--alpha-color-fg-black-darker),
+      light-grey: var(--alpha-color-fg-black-dark),
+    );
+
+    @each $button-color, $icon-color in $icon-color-map {
+      &:where(.color-#{$button-color}) {
+        :where(.ButtonIcon) {
+          color: $icon-color;
+        }
+      }
+    }
+  }
+
+  &:where(.variant-tertiary.color-white) {
+    :where(.ButtonIcon) {
```

```suggestion
    &:where(.ButtonIcon) {
```

전반적으로 통일하는 게 일관성 있을 거 같아요

---

```css
& :where(.ButtonIcon) {
```
로 하자는 말씀이시죠?? `& `처럼 띄어쓰기가 있으면 child 를 선택하는 걸로 이해했습니다 (없는 것과 마찬가지)

---

아 그렇네요, 잘못 봤네요 😅